apps/backoffice/src/lib/be/__tests__/institutions.test.ts

```diff
@@ -98,20 +101,27 @@ const mocks: {
   },
 }));
 
+// Define institutionProducts outside of vi.hoisted() to use imported functions
+const institutionProducts = getMockInstitutionProducts(
+  "id1",
+) as unknown as InstitutionProducts;
+
```

Comment:
suggestion: move inside test case

Since you are using this `const` only once, I suggest to move it directly into the test case that use it